_maps/RandomRuins/SpaceRuins/old_infiltrator.dmm

```diff
@@ -720,8 +717,8 @@
 /area/ruin/space/unpowered)
 "YL" = (
 /obj/structure/rack,
-/obj/item/clothing/suit/space/syndicate/black/red,
-/obj/item/clothing/head/helmet/space/syndicate/black/red,
+/obj/item/clothing/suit/space/syndicate/black/green,
+/obj/item/clothing/head/helmet/space/syndicate/black/green,
```

Comment:
I'll echo my statement from the Faceoff, I think breaking theme is worse than not including. This suit was in the same spot as the actual softsuit from the old infiltrator. Why not change it so that this corner has a hole blown out of it? You're already, I believe, editing these with StrongDMM.

Reply:
I'm not sure this breaks the theme, though - the only issue is that it's green, not red. It's still a syndicate suit. I could still delete it and flavor that it was blown up, but I don't want to take away something cool and not leave something else. I might do this and put the contractor suit somewhere else on the infiltrator.

Reply:
Sometimes you can't compromise perfectly. On that note, please don't let anyone see that suit, it needs help badly.

Also it is tiny so it can fit in pockets.